### PR TITLE
New package: mt-st-1.1

### PR DESCRIPTION
**Pull request by allan**

### CODE REVIEW

**Comment by xtraeme** [comment]
The installation stage is not installing anything into DESTDIR, please fix.

**Comment by xtraeme** [comment]
aka the package is completely empty.

**Comment by allan** [comment]
that one again... fixed.

File: srcpkgs/mt-st/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+maintainer="allan <mail@may.mooo.com>"
+license="GPL-2"
+homepage="http://ftp.ibiblio.org/pub/linux/system/backup/"
+distfiles="ftp://ftp.ibiblio.org/pub/linux/system/backup/mt-st-${version}.tar.gz"
+checksum=945cb4f3d9957dabe768f5941a9148b746396836c797b25f020c84319ba8170d
+
+do_install() {
+	vbin mt
+	vman mt.1
+	vinstall stinit 755 usr/sbin
```
Comment (xtraeme) [inline srcpkgs/mt-st/template]: Please use vbin here too, we are moving away from sbin.

**Pull request merged**